### DISCUSSION TITLE
Problem with Clio IV and DDT4ALL - Please Help me - Thanks very much

### DISCUSSION THREAD

**Discussion by alan3469** [cedricp/ddt4all#865] [Q&A]
Hello, please can you help me, I have disconnect the start and stop but I have made some mistake and the buzzer for the light when I take the key don't work and the rear left light don't work, stop ok. Please can you help me. Can I've made a reset of all setting or just fix my problem. thanks a lot !!

**Comment by matzel82** [comment]
It's sounds like a fuse ?!?

**Reply by alan3469** [reply]
no, I check. I've play with ddt4all to remove the start and stop I should never have,

**Comment by AndyIreland** [comment]
you mean when you walk away from car the bleep bleep does not sound?? (funny if it has just by disabling stop and start) 

and your light not working which one the brake light? / blinker light? / tail light? (again weird if disabling stop and start caused that) 

Download PYREN , let it load your ECU's - go into lighting ECU (should be EMM) and check the stored fault codes instead of guessing, see if any failures show up there / 'present'

**Reply by alan3469** [reply]
Thank you for your answers ! Were I canI download  Pyren? does it work with obdlink sx cable?

No, when I forget to turn off the high beams when I get out of the car, I no longer have the buzzer.
And the left rear light of the car no longer works.
Today I noticed that the fuel gauge remained with full.
It's not when I disabled the start and stop before I got to tweaking things in AlterinManagement.

**Reply by AndyIreland** [reply]
simpler version to download pyren is from here as it includes database in the app and places a shortcut on the desktop:
choose 184 installer for use with windows 

I dont know if works with obdlink sx cable - I use a elm327 with a PIC18F25K80 chip inside it
